### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by james-nash**
Found this branch that @byronwong started a long time ago to remove a load of unused images from the source code.

Figured it's worth keeping, so I've finished what he started and raised this PR.

Note: A Sonarwhal test (https://sonarwhal.com/scanner/56e495ec-9c49-4ac2-9e20-07b78e3c8890) alerted me to the fact that we had broken & redundant `apple-touch-icon` links in the `<head>`, so I fixed that while I was at it.

While fixing that, I also noticed that the OGP tags were referencing the old `buildit.digital` URL _sigh_, so that's fixed too.

### CODE REVIEW

File: src/includes/head.pug [hunk 1]
```diff
@@ -6,20 +6,16 @@ head
...
 
   meta(property='og:locale' content='en_GB')
   meta(property='og:type' content='article')
   meta(property='og:title' content='buildit@wiprodigital')
   meta(property='og:description' content='The future belongs to those who make')
-  meta(property='og:url' content='https://buildit.digital/')
-  meta(property='og:site_name' content='buildit_')
-  meta(property='og:image' content='https://buildit.digital/assets/images/buildit-logo-white.png')
-  meta(property='article:published_time' content='Fri March 17, 2017')
+  meta(property='og:url' content='https://buildit.wiprodigital.com/')
```
Comment (ThePeach) [inline src/includes/head.pug]: this sheet needs to be parametrised!!!

File: src/includes/head.pug [hunk 2]
```diff
@@ -6,20 +6,16 @@ head
   link(rel='shortcut icon' href='assets/images/favicon.png' type='image/png')
   link(rel='icon' href='assets/images/favicon.png' type='image/png')
   link(rel='apple-touch-icon' href='assets/images/apple-touch.png')
-  link(rel='apple-touch-icon' sizes='72x72' href='assets/images/apple-touch-72.png')
-  link(rel='apple-touch-icon' sizes='114x114' href='assets/images/apple-touch-114.png')
-  link(rel='apple-touch-icon' sizes='144x144' href='assets/images/apple-touch-144.png')
-  
+
```
Comment (ThePeach) [inline src/includes/head.pug]: any reason for ditching these icons? just wondering

File: src/includes/head.pug (hunk 1 above)
Comment (james-nash) [reply src/includes/head.pug]: It _should_ be, yeah. I don't think it's worth trying to fix that at this point though - let's just make sure the new site gets it right :-)

File: src/includes/head.pug (hunk 2 above)
Comment (james-nash) [reply src/includes/head.pug]: According to Sonarwhal's advice it's redundant to specify all the different sizes - just one, high-res icon will suffice. See https://sonarwhal.com/docs/user-guide/rules/apple-touch-icons/#page-heading That makes sense to me, so I removed them.

**Pull request merged**